src/lib/css-parse.html

```diff
@@ -89,6 +90,18 @@
       return node;
     },
 
+    // conversion of sort unicode escapes with spaces like `\33 ` (and longer) into
+    // expanded form that doesn't require trailing space `\000033`
+    _expandUnicodeEscapes : function(s) {
+      return s.replace(/\\([0-9a-f]{1,6})\s/gi, function() {

```

Comment:
`\s` matches more than a trailing space (tabs, spaces and new-lines). Is that intentional? If you only want to match a trailling space, it should be `/\\([0-9a-f]{1,6}) /gi`